GX-AnalyticContinuation/ABINIT/m_simple_pade.f90

```diff
@@ -0,0 +1,205 @@
+module m_simple_pade
+
+  implicit none
+
+  public :: pade
+
+  public
+  integer, parameter :: dp=kind(1.0d0)
+  integer, parameter :: dpc=kind((1.0_dp,1.0_dp))
```

Comment:
I don't understand what dp and dpc are. I've opened merge #18 that defines the precision in a sensible place but I need to finish it off